apprise/plugins/NotifyTwilio.py

```diff
@@ -108,7 +108,7 @@ class NotifyTwilio(NotifyBase):
             'type': 'string',
             'private': True,
             'required': True,
-            'regex': (r'^[a-f0-9]+$', 'i'),
+            'regex': (r'^[a-z0-9]+$', 'i'),
```

Comment:
Though account auth tokens only use A-F, the auth tokens generated for API Keys seem to use A-Z

Reply:
I'm guessing that if this is being restored to handle auth tokens, it can be reverted back?  You've already updated the regex for the new API Key.  Thoughts?

Reply:
Sorry, my original comment wasn't very clear...

- Account-wide auth tokens (which previously were the only type of token available) are used with the Account SID to authenticate. These tokens seem to only be `a-f`
- API auth tokens (also called API Secrets in [these docs](https://www.twilio.com/docs/usage/api#authenticate-with-http)) are used with the API key to authenticate. These tokens use `a-z`

After this PR, users can pass:
- `twilio://AccountSid:AuthToken@FromPhoneNo`
- `twilio://AccountSid:APISecret@FromPhoneNo?apikey=ApiKey`

Since both types of token/secret are passed in the second position of the `twilio://` scheme, we have to relax the regex to accept both types